services/shhext/api.go

```diff
@@ -54,10 +54,13 @@ type MessagesRequest struct {
 
 func (r *MessagesRequest) setDefaults(now time.Time) {
 	// set From and To defaults
-	if r.From == 0 && r.To == 0 {
-		r.From = uint32(now.UTC().Add(-24 * time.Hour).Unix())
+	if r.To == 0 {
 		r.To = uint32(now.UTC().Unix())
 	}
+
+	if r.From == 0 {
+		r.From = r.To - 86400 // -24 hours
```

Comment:
it might be a valid request to receive **all** history for a chat where you just joined

Reply:
@dshulyak you are right, the problem is that I can't understand if `0` is the zero value of that field or manually set to zero. Maybe the client can ask from `1`, or any date far in the past. would it make sense?

Reply:
What's gonna happen if `From` is set to `0`? `To` should not be `0` ever.